js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_pt_BR.properties

```diff
@@ -505,7 +505,7 @@ policyEnforcementModes.ENFORCING=Restritiva
 debug=Debug
 clear=Limpar
 resources=Recursos
-groups=Groups user is a member of. Select a listed group and click the Leave button to leave the group.
+groups=Grupos.
```

Comment:
@marisvaldocopilot8 Thank you for the PR. Could you please remove the comma here?